src/vs/workbench/browser/parts/titlebar/media/titlebarpart.css

```diff
@@ -6,7 +6,6 @@
 .monaco-workbench > .part.titlebar {
 	box-sizing: border-box;
 	width: 100%;
-	font-size: 12px;
```

Comment:
I don't think we want to make the title bigger, it currently is very close to the native size.